### ISSUE TITLE
Algolia should search organization descriptions

### ISSUE THREAD

**Issue by jasonLaster** [climatescape/climatescape.org#70]
I believe the algolia search index has the description field, but does not search it.

**Comment by bloudermilk** [comment]
@jasonLaster I'm not sure how I see this working without both being able to show description snippets in search results and being able to view an exhaustive paginated list of results.

**Comment by jasonLaster** [comment]
Yeah - it is a bit fuzzy... I was imagining ranking it fairly low, so a name match was given a lot more weight.

We can close this or punt. not a big deal.
